### PR TITLE
Use relevant CowSwap quote API instead of legacy

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

https://cowservices.slack.com/archives/C0375NV72SC/p1678361727279049?thread_ts=1678277967.637559&cid=C0375NV72SC

The `/markets` endpoint was deleted, so `getPriceQuoteLegacy` method from SDK doesn't work.
In this PR the legacy method was replaced by the relevant CowSwap quote API.

### CODE REVIEW

File: src/custom/utils/price.ts [hunk 1]
```diff
@@ -86,7 +86,23 @@ export type AllPricesResult = {
...
+    },
+  }).then(([result]) => {
+    if (result.status === 'fulfilled') {
+      return result.value
+    }
+
+    return Promise.reject(result.reason)
+  })
+
+  const gpPricePromise = withTimeout(cowSwapQuote, PRICE_API_TIMEOUT_MS, 'GPv2: Get Price API')
```
Comment (anxolin) [inline src/custom/utils/price.ts:105]: nit: I think we dont't need the GPv2 name any more. Can be named CoW Protocol

**Review by anxolin — APPROVED** [review]
Approved. In principle the PR can be done against hotfix/1.30.1 since it doesn't depend on 1inch thingy. But don't recreate. Its fine..

**Comment by shoom3301** [comment]
>In principle the PR can be done against hotfix/1.30.1 since it doesn't depend on 1inch thingy
Ok, we can cherry-pick it if any

File: src/custom/utils/price.ts (hunk 1 above)
Comment (shoom3301) [reply src/custom/utils/price.ts]: Fixed

**Comment by elena-zh** [comment]
Hey @shoom3301 , Fee amount is missing on the form when fees exceed from amount
image

**Comment by elena-zh** [comment]
I can see message 'Fees exceeds from amount', however, the trade is precalculated, fees are 90% of the current amount:
image

So I expect here whether enabled Swap button, whether this screen 
image

**Comment by shoom3301** [comment]
Actually, this PR doesn't make sense. The fallback to the legacy strategy assumes that the COWSWAP strategy doesn't work hence we shouldn't call it.

**Pull request closed without merging**